### PR TITLE
Announcemnt blog for ADB 1.7.0

### PR DESCRIPTION
**Pull request by LalatenduMohanty**
Signed-off-by: Lalatendu Mohanty <lmohanty@redhat.com>

### CODE REVIEW

File: source/blog/2016-02-17-ADB-1-7.html.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+categories:
+- Blog
+---
+Project Atomic is pleased to announce the release of the Atomic  Developer Bundle (ADB) 1.7.0, a Vagrant box that provides a ready-to-use  development environment for container applications. With the ADB,  developers can dive right into producing complex, multi-container  applications. The ADB is useful on Microsoft Windows, Apple’s OS X and  GNU/Linux distributions.
+
+The ADB provides a container development ecosystem that ensures that  developers are able to work with Linux containers even if their base operating system does not support it.
+The ADB provides a quick and easy way to setup multi-container  applications that take advantage of management and/or orchestration  solutions, such as OpenShift or Kubernetes. The ADB also supports [Nulecule](http://www.projectatomic.io/docs/nulecule/) and  [Atomic App](http://www.projectatomic.io/docs/atomicapp/).  These projects provide a structure for packaging, distributing, and deploying multi-container applications.
+
+The 1.7.0 release adds several new features, including:
+ * OpenShift is now managed as a systemd unit file with full functionality [start/stop/restart].
```
Comment (bexelbie) [inline source/blog/2016-02-17-ADB-1-7.html.md]: We should explain why this is a good thing for users.

File: source/blog/2016-02-17-ADB-1-7.html.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+ * We have a new cli i.e. sccli inside the box which can help users move between Kubernets and OpenShift setup.
```
Comment (bexelbie) [inline source/blog/2016-02-17-ADB-1-7.html.md]: Kubernetes is misspelled.  Should CLI be capitalized?  sccli??

File: source/blog/2016-02-17-ADB-1-7.html.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+ * We have a new Vagrant plugin i.e. [vagrant-service-manager](https://github.com/projectatomic/vagrant-service-manager/) to manage services inside the ADB Vagrant box .  This is the recomended plugin inplace of vagrant-adbinfo now.
```
Comment (bexelbie) [inline source/blog/2016-02-17-ADB-1-7.html.md]: Suggested:

* We have a new vagrant plugin, vagrant-service-manager, that manages the services inside of the ADB.  This plugin is recommended for use with the ADB and fully replaces vagrant-adbinfo.

File: source/blog/2016-02-17-ADB-1-7.html.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+Additional Resources and Download Locations
```
Comment (bexelbie) [inline source/blog/2016-02-17-ADB-1-7.html.md]: This header doesn't seem to go with the paragraph that follows.   What about deleting this header line and ... (see below)

File: source/blog/2016-02-17-ADB-1-7.html.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+Project Atomic integrates the tools and patterns of container-based application and service deployment with trusted operating system platforms to deliver an end-to-end hosting architecture that's modern, reliable and secure.
```
Comment (bexelbie) [inline source/blog/2016-02-17-ADB-1-7.html.md]: starting this line with "The ADB is built as part of Project Atomic. "

File: source/blog/2016-02-17-ADB-1-7.html.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+Interested in learning more about the ADB, or Project Atomic?
```
Comment (bexelbie) [inline source/blog/2016-02-17-ADB-1-7.html.md]: Interested in downloading or learning more ...

File: source/blog/2016-02-17-ADB-1-7.html.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+ * Read more about the Atomic Developer Bundle (ADO) at the project's [github.com page](https://github.com/projectatomic/adb-atomic-developer-bundle), including a quick start guide.
+ * The ADB can be downloaded in [multiple ways](https://github.com/projectatomic/adb-atomic-developer-bundle/blob/master/docs/installing.rst#3-download-the-adb)
+ * [Project Documentation](https://github.com/projectatomic/adb-atomic-developer-bundle#documentation) and [Installation Guide](https://github.com/projectatomic/adb-atomic-developer-bundle/blob/master/docs/installing.rst)
+ * Get more Project Atomic updates on Twitter [@ProjectAtomic](https://twitter.com/projectatomic)
```
Comment (bexelbie) [inline source/blog/2016-02-17-ADB-1-7.html.md]: s/more//

File: source/blog/2016-02-17-ADB-1-7.html.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+ * We have a new Vagrant plugin i.e. [vagrant-service-manager](https://github.com/projectatomic/vagrant-service-manager/) to manage services inside the ADB Vagrant box .  This is the recomended plugin inplace of vagrant-adbinfo now.
+
+Additional Resources and Download Locations
+
+Project Atomic integrates the tools and patterns of container-based application and service deployment with trusted operating system platforms to deliver an end-to-end hosting architecture that's modern, reliable and secure.
+
+Interested in learning more about the ADB, or Project Atomic?
+
+ * Read more about the Atomic Developer Bundle (ADO) at the project's [github.com page](https://github.com/projectatomic/adb-atomic-developer-bundle), including a quick start guide.
+ * The ADB can be downloaded in [multiple ways](https://github.com/projectatomic/adb-atomic-developer-bundle/blob/master/docs/installing.rst#3-download-the-adb)
```
Comment (bexelbie) [inline source/blog/2016-02-17-ADB-1-7.html.md]: Should we include a pointer to at least one way?

**Comment by jberkus** [comment]
I had to do a lot of edits on this, so it was simpler to fork it to my own PR:  #233

Please continue discussion on that PR.  Thanks!

**Comment by jberkus** [comment]
Closing this in favor of revised version.

**Pull request closed without merging**